README.md

```diff
@@ -15,6 +15,7 @@ buildscript {
   }
   dependencies {
     classpath 'com.vanniktech:gradle-maven-publish-plugin:0.13.0'
+    classpath 'org.jetbrains.dokka:dokka-gradle-plugin:1.x.x'
```

Comment:
Can you add a comment that it's only needed for Kotlin projects?

Reply:
Sure